.scripts/build_steps.sh

```diff
@@ -34,9 +34,9 @@ CONDARC
 export CONDA_LIBMAMBA_SOLVER_NO_CHANNELS_FROM_INSTALLED=1
 
 mamba install --update-specs --yes --quiet --channel conda-forge --strict-channel-priority \
-    pip mamba conda-build conda-forge-ci-setup=4
+    pip mamba conda-build boa conda-forge-ci-setup=4
 mamba update --update-specs --yes --quiet --channel conda-forge --strict-channel-priority \
-    pip mamba conda-build conda-forge-ci-setup=4
+    pip mamba conda-build boa conda-forge-ci-setup=4
```

Comment:
Let's hope that the interaction of `boa` with `conda-forge-ci-setup=4` goes smoothly (first time we're trying this AFAIK) 🤞

Reply:
It's been tested at https://github.com/conda-forge/cf-autotick-bot-test-package-feedstock/pull/489

Reply:
I mean broadly across many feedstocks. There were a bunch of incompatibilities between mamba, boa & conda-build, let's hope they all got fixed in the recent round of releases and that no others come up.